cli/main.go

```
@@ -43,10 +44,42 @@ func buildVersionString() string {
 			fmt.Fprintf(versionsWriter, "\t%s\n", image)
 		}
 	}
-	if refValues, err := json.MarshalIndent(manifest.GetEmbeddedReferenceValues(), "\t", "  "); err == nil {
-		fmt.Fprintf(versionsWriter, "embedded reference values:\t%s\n", refValues)
+
+	embeddedReferenceValues := manifest.GetEmbeddedReferenceValues()
+	for _, platform := range platforms.All() {
+		fmt.Fprintf(versionsWriter, "\nreference values for %s platform:\n", platform.String())
+
+		runtimeHandlerName, err := manifest.RuntimeHandler(platform)
+		if err != nil {
+			panic(fmt.Sprintf("Couldn't get runtime handler name for platform %s", platform.String()))
```

Why are we panicking here instead of returning the error?